recipes/netket/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "netket" %}
+{% set version = "2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/netket/netket/archive/v2.1.tar.gz
+  sha256: 041d2b058c5d2945bf0a4194ad2bf3c426ad9b6ce0dd323a81a7154bc6e45452
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv --install-option=--cmake-args='-DNETKET_USE_LAPACK=OFF -DCMAKE_BUILD_TYPE=Release -DNETKET_USE_OPENMP=ON -DNETKET_USE_BLAS=ON -DNETKET_USE_SLEEF=ON'"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake >=3.12
+    - ninja
+  host:
+    - pip
+    - python =3.7
+    - mpich
+    - libblas
+    - libcblas
+    - openmp  # [linux]
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
+    - libcxx
+  run:
+    - numpy >=1.16
+    - python =3.7
+    - scipy >=1.2.1
+    - numba >=0.48
+    - mpich
+    - mpi4py
+    - tqdm
+    - libblas
+    - libcblas
+    - libcxx
+    - libgomp  # [linux]
+    - openmp  # [linux]
+    - llvm-openmp  # [osx]
```

Comment:
you can remove the 6 lines above

Reply:
the 6 lines above are 
```
     - libblas		
     - libcblas		
     - libcxx		
     - libgomp  # [linux]		
     - openmp  # [linux]		
     - llvm-openmp  # [osx]		
```
maybe I should keep libcblas? Having removed all of them causes a build failure on linux

Reply:
see https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=147811&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0&l=1357